docs/how-to-guides/use-lambda-functions.md

```diff
@@ -0,0 +1,223 @@
+# Use lambda functions
```

Comment:
Add YAML front matter (just crib from another md file).

Reply:
Where would I find it? It doesn't look like any of the other how-to guides have it unless my IDE is just hiding it

Reply:
Heh, well look at that! Every page should have yaml -- go ahead and merge and I'll add to  the frontmatter to all of these at once.